oarepo_vocabularies/ui/theme/assets/semantic-ui/js/oarepo_vocabularies_ui/form/components/LocalVocabularySelectField/LocalVocabularySelectField.jsx

```diff
@@ -130,36 +109,50 @@ export const LocalVocabularySelectField = ({
       processVocabularyItems(featuredOptions, showLeafsOnly, filterFunction),
     [featuredOptions, showLeafsOnly, filterFunction]
   );
-  const hasMultipleItems = multiple || fieldData.detail === "array";
 
   const handleChange = ({ data, formikProps }) => {
     if (hasMultipleItems) {
```

Comment:
Could this func be extracted into some utility func? Very similar thing used by VocabSelectField